libraries/AP_BoardConfig/AP_BoardConfig.cpp

```diff
@@ -143,7 +151,7 @@ const AP_Param::GroupInfo AP_BoardConfig::var_info[] = {
     // @Bitmask: 0:Ch1,1:Ch2,2:Ch3,3:Ch4,4:Ch5,5:Ch6,6:Ch7,7:Ch8,8:Ch9,9:Ch10,10:Ch11,11:Ch12,12:Ch13,13:Ch14
     // @RebootRequired: True
     // @User: Advanced
-    AP_GROUPINFO("SAFETY_MASK", 7, AP_BoardConfig, px4.ignore_safety_channels, 0),
+    AP_GROUPINFO("SAFETY_MASK", 8, AP_BoardConfig, px4.ignore_safety_channels, 0),
```

Comment:
you can't change the indexes for existing variables, or it will break peoples existing setups. The indexes are used inside FRAM for current parameter values
You'll just have to use later index numbers